test/run/ch-run_misc.bats

```diff
@@ -570,10 +570,10 @@ EOF
     for f in $files $files_optional; do touch "${img}/${f}"; done
 
     # This should start up the container OK but fail to find the user command.
-    run ch-run "$img" -- true
+    run ch-run "$img" -- /bin/true
```

Comment:
You'll need to change all the other `true` run in a container to `/bin/true` as well (e.g., line 583). Run on the host, no need (e.g., line 582).